packages/eslint-plugin-template/docs/rules/interactive-supports-focus.md

```diff
@@ -555,8 +555,8 @@ The rule does not have any configuration options.
 <button (click)="onClick()" class="foo">Foo</button>
 <option (click)="onClick()" class="foo">Food</option>
 <select (click)="onClick()" class="foo"></select>
-<summary (click)="onClick()">Foo</summary>
-<textarea (keypress)="onKeypress()" class="foo"></textarea>
+<summary (click)="onClick()" aria-expanded>Foo</summary>
+<textarea (keypress)="onKeypress()" class="foo" aria-multiline></textarea>
```

Comment:
The aria-multiline attribute should only be required if the textbox role were used to create a multiline textbox without a semantic HTML `<textarea>` element. The `<textarea>` element implies multiline support. http://udn.realityripple.com/docs/Web/Accessibility/ARIA/Roles/textbox_role